README.md

```diff
@@ -1 +1,51 @@
-# Setup-and-Mobile-Version-skeleton
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+## Setup and Mobile Version Skeleton
+
+
+
+> Description the project.
+
+The 2 sections of this skeleton project has:
+
+a - the toolbar
+b - the headline section
+
+
+
+## Built With
+
+- HTML
+- CSS
+- linter
+
+
+
```

Comment:
- [ ] Please follow the Readme file [template](https://github.com/microverseinc/readme-template/blob/master/README.md) as the other reviewer requested you to do. Kindly add at least the getting started section to let us know how to clone, setup, and run the project. Keep rocking 💪